modules/UI/toolbars/Toolbar.js

```diff
@@ -633,26 +635,28 @@ const Toolbar = {
     },
 
     /**
-     * Marks audio icon as disabled or not.
-     * @param {boolean} disabled if icon should look like disabled or not
+     * Enables / disables audio toolbar button.
+     *
+     * @param {boolean} enabled indicates if the button should be enabled
+     * or disabled
      */
-    disableAudioIcon (disabled) {
+    setAudioIconEnabled (enabled) {
         var $btn = $('#toolbar_button_mute');
 
         $btn
-            .prop("disabled", disabled)
-            .attr("data-i18n", disabled
-                ? "[content]toolbar.micDisabled"
-                : "[content]toolbar.mute")
-            .attr("shortcut", disabled ? "" : "mutePopover");
+            .prop("disabled", !enabled)
+            .attr("data-i18n", enabled
+                ? "[content]toolbar.mute"
+                : "[content]toolbar.micDisabled")
+            .attr("shortcut", enabled ? "mutePopover" : "");
 
-        disabled
-            ? $btn.attr("disabled", "disabled")
-            : $btn.removeAttr("disabled");
+        enabled
+            ? $btn.removeAttr("disabled")
+            : $btn.attr("disabled", "disabled");
 
         APP.translation.translateElement($btn);
 
-        disabled && this.toggleAudioIcon(disabled);
+        !enabled && this.toggleAudioIcon(!enabled);
```

Comment:
Having !enabled here bothers me, but the toggleAudioIcon takes muted as parameter, so I thought it makes sense to leave it like that.